docs/sources/flow/concepts/config-language/_index.md

```diff
@@ -42,6 +42,12 @@ BLOCK_NAME {
 }
 ```
 
+[River is designed][RFC] with the following requirements in mind:
+
+* _Fast_: The configuration language must be fast so the component controller can quickly evaluate changes.
+* _Simple_: The configuration language must be easy to read and write to minimize the learning curve.
+* _Debuggable_: The configuration language must give detailed information when there's a mistake in the configuration file.
+
 [River][RFC] is similar to HCL, the language Terraform and other Hashicorp projects use.
```

Comment:
I think we can drop the second link here to the RFC, readers might see a second link and mistake it for a new link.